### ISSUE TITLE
Autocomplete doesn

### ISSUE THREAD

**Issue by birbilis** [olifolkerd/tabulator#3045]
Using JSON autocomplete columns 

```
autoColumns: true,
autoColumnsDefinitions: {
     //...
    Municipality: {
                headerFilter: "autocomplete", //pick values
                headerFilterLiveFilter: true,
                headerFilterParams: {
                    showListOnEmpty: true,
                    allowEmpty: true,
                    freetext: true,
                    verticalNavigation: "hybrid",
                    values: true,
                    sortValuesList: "asc"
                },
                width: 300,
                download: true,
                titleDownload: "Δήμος"
            },
//...
}
```

even if I use headerFilterLiveFilter: true I can't make autocomplete to filter columns as I type (apart from offering me a list of values to pick / hint me on what is available)

**Comment by birbilis** [comment]
the docs about Real Time Filtering only seem to mention input editor
[http://tabulator.info/docs/4.8/filter](http://tabulator.info/docs/4.8/filter)

> Real Time Filtering
> 
> If an input element is used as the filter element, the table will be filtered in real time as the user types in the input element. To prevent exessive redrawing of the table Tabulator will wait 300 miliseconds after the user has finished typing before tirggering the filter, this ensures that the table is not redrawn for every character typed by the user.
> 
> Note: If the input has a type attribute value of text it will be automatically changed to search to give the user the option to clear the input text.
> 
> If you would prefer that the input element behave like a standard editor without live updating the table, you can set the headerFilterLiveFilter column definition property to false
> 
> {title:"Name", field:"name", headerFilter:"input", headerFilterLiveFilter:false}
> Live Filter Delay
> By default Tabulator will wait 300 miliseconds after a keystroke before triggering the filter. You can customise this delay by using the headerFilterLiveFilterDelay table setup option:
> 
> var table = new Tabulator("#example-table", {
>     headerFilterLiveFilterDelay:600, //wait 600ms from last keystroke before triggering filter
> });

**Comment by birbilis** [comment]
This may or may not be related to another issue I see with the autocomplete, that I can't (with the params mentioned above) type in and filter a partial value (as I can do with the input header filter). 

There is no live example on the doc site that uses autocomplete in header, only one that uses it in cell text [here](http://tabulator.info/examples/4.8?#editable) and that one isn't set to allow you to type in any value, nor is related to real-time filtering of course since it's not at the header. It would be nice to have an online sample to point to / try (so that I'd know it's not something with my params and see the suggested ones live)

**Comment by olifolkerd** [comment]
Hey @birbilis 

Could i ask that you take a minute to have a play with the component and get to know it before rasing soo many issues. it has been ten minutes and you have already raised 3 issues and commented on another.

If you want advice on how to use a particular feature please ask on StackOverflow. If you feel you have found a genuine bug then you need to use the [Bug Reporting Issue Template](https://github.com/olifolkerd/tabulator/issues/new?assignees=&labels=Possible+Bug&template=bug_report.md&title=) and provide a the required details and a JS Fiddle, bulk copy and pasting large chunks of the documentation in is extremely unhelpful and makes the issue very hard to parse.

Cheers

Oli :)

**Comment by birbilis** [comment]
tried adding

>                 headerFilterLiveFilterDelay:300, //wait 300ms from last keystroke before triggering filter

too to the autocomplete but that didn't help either

**Comment by birbilis** [comment]
This may be by design / not considered during design, but user will probably consider it a bug if they also have in the same grid columns that use the input header filter and some that have autocomplete (because the values are of limited count that makes sense to use such). The first ones will realtime filter, the others won't

**Comment by birbilis** [comment]
sorry for the multiple comments, I'm in a hurry to roll-out a COVID-19 related viz that uses it, can post a test URL here, but it will break eventually when the test page is removed

**Comment by olifolkerd** [comment]
@birbilis as mentioned above, please take a breather.

This constant stream of consciousness is not a helpful discussion and will not help resolve your issue. I understand you may be in a hurry, but spamming this issues list is not the way to get help.

If you have an issue and need help please post a concise question on Stack Overflow that outlines what you want to achieve and someone will be able to help you. At the moment all you are doing is spamming my inbox with notifications :(

Cheers

Oli :)

**Comment by birbilis** [comment]
Sorry for the spamming, thought you were using GitHub for related discussions, will move them to SO if there is a community already for tabulator there.

**Comment by olifolkerd** [comment]
Hey @birbilis 

This issues list is intended for Bug Reports and Feature Requests only as per the [Community Guidelines}(http://tabulator.info/community).

It would sadly be impossible to track the actual issues at hand if every developer started posting questions on here as well.

Thanks for your understanding

Cheers

Oli :)
